tests/dags/test_dag_parsing.py

```diff
@@ -18,12 +18,12 @@
     "providers/provider_workflow_dag_factory.py",
     "maintenance/airflow_log_cleanup_workflow.py",
     "maintenance/pr_review_reminders/pr_review_reminders_dag.py",
-    "commoncrawl/sync_commoncrawl_workflow.py",
-    "commoncrawl/commoncrawl_etl.py",
     "database/recreate_popularity_calculation_dag_factory.py",
     "data_refresh/dag_factory.py",
     "oauth2/authorize_dag.py",
     "oauth2/token_refresh_dag.py",
+    #    "commoncrawl/sync_commoncrawl_workflow.py", # retired
+    #    "commoncrawl/commoncrawl_etl.py", #retired
```

Comment:
These lines can also just be removed
```suggestion
```

Reply:
Hi @AetherUnbound Thanks for informing of these changes I'll open a new PR for the same.

Reply:
Hi @AetherUnbound , I just committed the changes you mentioned. Let me know if any other changes are required and I am fairly new at Open source  contributions in general so please feel free to let me know if any mistakes I made.